### ISSUE TITLE
About the Dismount-CimDiskImage error

### ISSUE THREAD

**Issue by jon0225** [JimMoyle/CimDiskImage-PowerShell#1]
We are currently testing app attach.
[https://learn.microsoft.com/ja-jp/azure/virtual-desktop/app-attach-test-msix-packages?tabs=posh6%2Ccimfs]

When I run Dismount-CimDiskImage, I get the following error
DismountingImage volume \\? \Volume{}} failed with error 'Access Denied'

What is the cause of this error and what is the solution?

Environment
windows11 enterprise 24H2
powershell 5.1

**Comment by JimMoyle** [comment]
Are you running as local admin?

**Comment by jon0225** [comment]
yes. I'm running it with local admin.

**Comment by JimMoyle** [comment]
Hmm, not sure then. If I encounter an error, I just pass it through from
the underlying API.

That's a Windows error, not one I wrote.

Jim

**Comment by JimMoyle** [comment]
You can see I'm giving you the return from the API results transcribed from
this doc.

https://learn.microsoft.com/en-us/previous-versions/windows/desktop/vdswmi/dismount-method-in-class-win32-volume

**Comment by breento** [comment]
I got the same error with Windows 11 24H2 while running as admin.

**Comment by jon0225** [comment]
When I mount it with a cim image and start it from the start menu, I get an error message saying "Element not found." Is this the same thing?

**Comment by JimMoyle** [comment]
@jon0225 I don't think so, this seems like a different error, have you tried with a vhdx?

**Comment by jon0225** [comment]
@JimMoyle 
I also tried vhdx. It worked without any problems.
Only with CimFS, an error occurs at startup and the mount cannot be removed.

**Comment by ounet** [comment]
@jon0225 and @JimMoyle with windows 11 24h2 CimFS image doesn't work. Microsoft issue : https://learn.microsoft.com/en-us/azure/virtual-desktop/app-attach-setup?tabs=portal&pivots=app-attach

**Comment by JimMoyle** [comment]
> @jon0225 and @JimMoyle with windows 11 24h2 CimFS image doesn't work. Microsoft issue : https://learn.microsoft.com/en-us/azure/virtual-desktop/app-attach-setup?tabs=portal&pivots=app-attach

This will be fixed via Microsoft update in June
